newdle/client/src/components/creation/CreationHeader.js

```diff
@@ -11,7 +11,8 @@ export default function CreationHeader() {
   const steps = [
     {
       title: 'Choose the participants',
-      description: 'You can also "Skip" this step and add them later',
+      description:
+        'You can also "Skip" this step and add participants later by sharing link to your Newdle',
```

Comment:
```suggestion
        'You can also "Skip" this step and simply share the link to your newdle',
```

Reply:
I'd then suggest to modify the "Finalize your Newdle" to "Finalize your new newdle" to keep it consistent

Reply:
"new newdle" sounds pretty strange ;o